content/v1/features/unity.md

```diff
@@ -465,4 +465,30 @@ The user can also set the volume limit for all the nodes in the cluster by speci
 ## Log Levels
 The CSI Driver for Dell EMC Unity allows users to configure different log levels using **logLevel** parameter. 
 
-The **logLevel** parameter needs to be configured in unity-creds secret created from secret.json or secret.yaml. The supported log levels are Info/Debug/Warn/Error. The default level is Info if logLevel is not configured by the user. The parameter can be changed dynamically without the need of driver re-installation or restart.
\ No newline at end of file
+The **logLevel** parameter needs to be configured in unity-creds secret created from secret.json or secret.yaml. The supported log levels are Info/Debug/Warn/Error. The default level is Info if logLevel is not configured by the user. The parameter can be changed dynamically without the need of driver re-installation or restart.
+
+## Single Pod Access Mode for PersistentVolumes
+Starting in version 2.1, CSI Driver for Unity now supports a new accessmode `ReadWriteOncePod` for PersistentVolumes and PersistentVolumeClaims. With this feature, CSI Driver for Unity allows to restrict volume access to a single pod in the cluster
+
+Prerequisites
+1. Enable the ReadWriteOncePod feature gate for kube-apiserver, kube-scheduler, and kubelet as the ReadWriteOncePod access mode is in alpha for Kubernetes v1.22 and is only supported for CSI volumes. You can enable the feature by setting command line arguments:
+`--feature-gates="...,ReadWriteOncePod=true"`
+2. Create a PVC with access mode set to ReadWriteOncePod like shown in the sample below
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: single-writer-only
+spec:
+  accessModes:
+  - ReadWriteOncePod # Allow only a single pod to access single-writer-only.
+  resources:
+    requests:
+      storage: 1Gi
+```
+
+## Volume Health Monitoring
+Starting in version 2.1, CSI Driver for Unity now supports volume health monitoring. This feature
+1. Reports on the condition of the underlying volumes via events when a volume condition is abnormal.
+2. Collects the volume stats.
+3. If feature gate `CSIVolumeHealth` is set to true, it also logs events to pods if volume condition is abnormal.   
```

Comment:
can we mention the events will be visible in controller logs for clear understanding? (how user able to view these events)